src/app/pages/send-tokens/components/asset-search/selected-asset-item.tsx

```diff
@@ -16,7 +16,7 @@ export const SelectedAssetItem = memo(
     const [field] = useField('assetId');
     const { name, selectedAsset, ticker } = useSelectedAsset(field.value);
     if (!selectedAsset) return null;
-    const isStx = name === 'Stacks Token';
+    const isStx = selectedAsset.isNativeAsset;
```

Comment:
Yes, but there is also a `blockchain` prop, so when we go back to refactor the send form it will be easy to distinguish and also scale if other blockchains with assets are added. Changes to the send form code are just temporary bc it will be refactored soon.